app/views/devise/mailer/password_change.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Hello <%= @resource.email %>!</p>
+
+<p>We're contacting you to notify you that you're password has been changed.</p>
```

Comment:
I was unsure about the copy here, was hoping someone would make such a suggestion :)